redirect/routes.php

```diff
@@ -2,617 +2,613 @@
 
 class Route {
     private $routes = [
```

Comment:
Adding a comment here to indicate the invariant would be also useful.

Reply:
Good idea!